.travis.yml

```diff
@@ -18,8 +18,10 @@ before_install:
 - echo HRNGDEVICE=/dev/urandom | sudo tee /etc/default/rng-tools
 - sudo /etc/init.d/rng-tools restart
 install:
-- pip install ansible==1.8.4
-- pip install coveralls
+- pip freeze -l
+# Preinstalled travis pytest is not tracked by us
+- pip uninstall pytest -y
+- pip install -r testinfra/requirements.txt
```

Comment:
I don't agree that we should shoehorn these pip dependencies into `test-requirements`. The test requirements are already problematic in that they get installed on target hosts (such as app-staging) simply to run the tests. We haven't had a "developer requirements" for some odd reason, and the testinfra requirements here are at least close to that, in that they install required packages such as ansible, rather than merely _recommending_ a specific version of Ansible in the docs.

I wouldn't be opposed to moving these requirements into the requirements directory with the others, and calling them `dev-requirements` or similar, but I figured we could shake that out later.